linux-headers/pkg.yaml

```diff
@@ -34,11 +34,13 @@ steps:
             ;;
         esac
 
-        make -j $(nproc) INSTALL_HDR_PATH=dest headers_install ARCH=${arch}
+        make -j $(nproc) headers ARCH=${arch}
```

Comment:
headers_install now relies on `rsync` which we don't have: http://www.linuxfromscratch.org/lfs/view/development/chapter06/linux-headers.html

Reply:
Is this a workaround for 'installing' it?